keepalived.te

```diff
@@ -31,6 +34,7 @@ allow keepalived_t self:netlink_netfilter_socket create_socket_perms;
 allow keepalived_t self:netlink_route_socket nlmsg_write;
 allow keepalived_t self:packet_socket create_socket_perms;
 allow keepalived_t self:rawip_socket create_socket_perms;
+allow keepalived_t keepalived_tmp_t:file { create_file_perms write_file_perms };
```

Comment:
This line should be removed and used manage_files_pattern macro like on line 39.